tests/Tests/500__jupyterhub/minimal-pytorch-test.robot

```diff
@@ -34,6 +34,19 @@ PyTorch Image Workload Test
     Capture Page Screenshot
     JupyterLab Code Cell Error Output Should Not Be Visible
 
+Tensorboard Workload Test
+    [Documentation]  Verifies that tensorboard is accessible
+    [Tags]  Sanity
+    ...     PLACEHOLDER
+    ...     ODS-1414
+    Close Previous Server
+    # For KFNBC the environment variable should TENSORBOARD_PROXY_URL=/notebook/<nb-namespace>/<nb-name>/proxy/6006/
+    &{tensorboard_proxy} =  Create Dictionary  TENSORBOARD_PROXY_URL=/user/${TEST_USER.USERNAME}/proxy/6006/
+    Spawn Notebook With Arguments  image=${NOTEBOOK_IMAGE}  size=Default  envs=&{tensorboard_proxy}
+    Run Keyword And Ignore Error  Clone Git Repository And Run  https://github.com/redhat-rhods-qe/ods-ci-notebooks-main  
+    ...    ods-ci-notebooks-main/notebooks/500__jupyterhub/tensorboard/pytorch/tensorboard_profiling_pytorch.ipynb  10m
+    Page Should Contain Element    xpath://iframe[contains(@id, "tensorboard-frame")]
```

Comment:
should we check that the iframe contains something? a logo, a particular menu for example. Something which can tell us the iframe is not empty

Reply:
I'll double check the behaviour when tensorboard doesn't load

Reply:
You're right, even when it doesn't load it still has an iframe with id `tensorboard-frame-...`. I'll update this.

Reply:
Done